x-pack/test/api_integration/apis/secops/network_top_n_flow.ts

```diff
@@ -8,98 +8,227 @@ import expect from 'expect.js';
 import { networkTopNFlowQuery } from '../../../../plugins/secops/public/containers/network_top_n_flow/index.gql_query';
 import {
   GetNetworkTopNFlowQuery,
+  NetworkTopNFlowDirection,
   NetworkTopNFlowType,
 } from '../../../../plugins/secops/public/graphql/types';
-
 import { KbnTestProvider } from './types';
 
+const EDGE_LENGTH = 10;
+
 const networkTopNFlowTests: KbnTestProvider = ({ getService }) => {
   const esArchiver = getService('esArchiver');
   const client = getService('secOpsGraphQLClient');
-
   describe('Network Top N Flow', () => {
-    before(() => esArchiver.load('filebeat/default'));
-    after(() => esArchiver.unload('filebeat/default'));
+    describe('With filebeat', () => {
+      before(() => esArchiver.load('filebeat/default'));
+      after(() => esArchiver.unload('filebeat/default'));
+
+      const FROM = new Date('2019-02-09T01:57:24.870Z').valueOf();
+      const TO = new Date('2019-02-12T01:57:24.870Z').valueOf();
```

Comment:
Thanks for updating these integration tests! I think this was the last of them too! Maintenance is going to be a bit nicer now :)

Reply:
I set the other ones to have their FROM and TO be something way in the past and something way in the future so that as the archiver is run I won't have to worry about resetting the date time again. Instead just numbers of items.

No changes requested, just mentioning my thinking when I changed the others.